tutorials/polkadot-sdk/parachains/build-custom-pallet/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Pallet Tutorials
+description: TODO
```

Comment:
missing desc

Reply:
This structure is just a temporary placeholder. I'm working on the new layout for tutorials